makeup/html/shared.css

```diff
@@ -1,3 +1,48 @@
+.back{
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    background-color: rgba(60, 60, 60, 0.4);
+}
+ .message-block{
+    width: 400px;
+    height: 200px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 150px;
+    background-color: #F8ECE0;
+    border: 1px solid #FF6000;
+    padding: 10px 15px 15px 15px;
+    display: grid;
+    grid-template-rows: 30px 100px 70px;
+}
+.message-cross{
+    margin-left: 388px;
+    color:#FF6000;
+}
+.fa-times{
+    color:#FF6000;
+}
+.message-text{
+    font-family: sans-serif, Roboto;
+    font-size: 19px;
+    text-align: justify;
+    color: black;
+}
+.btn-ok {
+    height: 50px;
+    width: 160px;
+    font-size: 18px;
+    border: 1px solid #FF6000;
+    background-color: #FF6000;
+    color: #FFFFFF;
+    margin-top: 20px;
+    margin-left: 240px;
+}
+.btn-ok:focus {
+    outline: 0 !important;
```

Comment:
Is important nessesary here? Is there another way?

Reply:
You are right. I fixed it